test/jdk/tools/jpackage/helpers/jdk/jpackage/test/PackageTest.java

```diff
@@ -556,7 +556,7 @@ private void verifyPackageInstalled(JPackageCommand cmd) {
                 if (PackageType.WINDOWS.contains(cmd.packageType())
                         && !cmd.isPackageUnpacked(
                                 "Not verifying desktop integration")) {
-                    new WindowsHelper.DesktopIntegrationVerifier(cmd);
+                    new WindowsHelper.DesktopIntegrationVerifier(cmd, null);
```

Comment:
I'd rather add a loop calling DesktopIntegrationVerifier() for all launchers than copy/paste the code in AdditionalLauncher.java

Reply:
Fixed. I also removed code from AdditionalLaunchers.